chore: update the goverance for member (#3450)
chaos-mesh/chaos-mesh: GOVERNANCE.md

    - Join community discussions (e.g. community meetings, Slack).
- Sponsored by at least 1 Chaos Mesh [maintainer or committer](https://github.com/chaos-mesh/chaos-mesh/blob/master/MAINTAINERS.md).

> Contributors that meet the above requirements will then be invited to the GitHub organization "Chaos Mesh" by sponsor, and there would be an announcement published in the slack channel [(#project-chaos-mesh)](https://slack.cncf.io/). Chaos Mesh members will be invited into the Chaos Mesh organization, which is recognized by the Chaos Mesh badge under the profile.

Reviewer: ```suggestion
Contributors that meet the above requirements will then be invited to the GitHub organization "Chaos Mesh" by a sponsor, and there would be an announcement published in the slack channel [(#project-chaos-mesh)](https://slack.cncf.io/). 
```
